pkg/edgehub/module_test.go

```diff
@@ -0,0 +1,102 @@
+package edgehub
+
+import (
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/beehive/pkg/core"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/pkg/common/util"
+)
+
+// coreContext is beehive context used for communication between modules
+var coreContext *context.Context
+
+// edgeHubModule is edgeHub implementation of Module interface
+var edgeHubModule core.Module
+
+//TestEdgeHub_Name is function that registers the module and tests whether the correct name of the module is returned
+func TestEdgeHub_Name(t *testing.T) {
```

Comment:
Camel naming please.

Reply:
What about `TestEdgeHubName()`?

Reply:
Interesting, but I just want to keep consistent. If we introduce `_`, then there will be many cases where developers want to use `_`.

Reply:
In Kubernetes, camel naming is always used.

Reply:
Fine. No issues. I'll convert everything to camel case then

Reply:
Thanks.